swagger/oauth.yaml

```diff
@@ -0,0 +1,89 @@
+---
+# This is a Swagger (swagger.org) specification fragment. It is
+# designed to be included into a top-level Swagger 'index', not
+# as a standalone document.
+
+endpoints:
+  request_access_token:
+    post:
+      operationId: request_access_token
+      tags:
+        - oauth2
+      parameters:
+        - in: formData
+          name: grant_type
+          required: true
+          type: string
+        - in: formData
+          name: client_id
+          required: true
+          type: string
+        - in: formData
+          name: client_secret
+          required: false
+          type: string
+          format: password
+        - in: formData
+          name: username
+          required: false
+          type: string
+        - in: formData
+          name: password
+          required: false
+          type: string
+          format: password
```

Comment:
This can also take a parameter of `refresh_token` (in place of `username` and `password`) if the grant_type is `refresh_token`.